node/bft/src/gateway.rs

```diff
@@ -524,7 +524,7 @@ impl<N: Network> Gateway<N> {
         // If the event was unable to be sent, disconnect.
         if let Err(e) = &result {
             warn!("{CONTEXT} Failed to send '{name}' to '{peer_ip}': {e}");
-            debug!("{CONTEXT} Disconnecting from '{peer_ip}' (unable to send)");
+            trace!("{CONTEXT} Disconnecting from '{peer_ip}' (unable to send)");
```

Comment:
```suggestion
            warn!("{CONTEXT} Disconnecting from '{peer_ip}' - Failed to send '{name}': {e}");
```

We can unify the 2 log messages to a single one that matches the other disconnect warn messages.